src/components/PrinterModal.vue

```diff
@@ -47,28 +51,40 @@ export default {
     clearSelect() {
       this.selectedPrinterId = null
     },
+    onPrintAction() {
+      this.isShow = true
+    },
     handleOk(evt) {
       // Prevent modal from closing
-      evt.preventDefault()
+      if (evt) {
```

Comment:
Instead of preventing the default behaviour here, vue has a neat thing where you can specify this behaviour in the component event declaration. https://vuejs.org/guide/essentials/event-handling.html#event-modifiers

Reply:
Yes , .prevent actions are ver handy. But it seems like we didn't need preventDefault at all in this case with the new changes, as we are dealing with those actions within the custom modal